server/tests/api/server/services.ts

```diff
@@ -88,7 +88,7 @@ describe('Test services', function () {
 
     const res = await getOEmbed(server.url, oembedUrl)
     const expectedHtml = '<iframe width="560" height="315" sandbox="allow-same-origin allow-scripts" ' +
-      `src="http://localhost:${server.port}/video-playlists/embed/${playlistUUID}" ` +
+      `title="${video.name}" src="http://localhost:${server.port}/video-playlists/embed/${playlistUUID}" ` +
```

Comment:
Should be `title="The Life and Times of Scrooge McDuck" since it's the playlist embed

Reply:
can i assign directly the string ?